src/explore-education-statistics-admin/src/components/form/FormEditor.tsx

```diff
@@ -204,6 +204,17 @@ const FormEditor = ({
       table: {
         contentToolbar: tableContentToolbar,
       },
+      link: {
+        decorators: {
+          addDataGlossaryAttributeToGlossaryLinks: {
+            mode: 'automatic',
+            callback: (url: string) =>
+              url.startsWith(process.env.PUBLIC_URL) &&
```

Comment:
Could this startsWith instead actually just be part of the regex?

Reply:
If so, I'm not sure how to do it.